dipy/segment/energy_mrf.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import division, print_function, absolute_import
+import numpy as np
+
+
+def total_energy(masked_image, masked_segmentation,
+                 mu, var, index, label, beta):
+
+    energytotal = log_likelihood(masked_image, mu, var, index, label)
+    energytotal += gibbs_energy(masked_segmentation, index, label, beta)
+
+    return energytotal
+
+
+def log_likelihood(img, mu, var, index, label):
```

Comment:
Yes, that's right, I just thought that something like `neg_log_likelihood` would be a better name for this function